src/Round.js

```diff
@@ -1,12 +1,24 @@
 const data = require('./data');
 const prototypeQuestions = data.prototypeData;
 const util = require('./util');
-const card = require('./Card');
-const deck = require('./Deck');
-const turn = require('./Turn');
+const Turn = require('./Turn');
 
 class Round {
-  constructor() {};
+  constructor(deck) {
+    this.deck = deck;
+    this.turns = 1;
+    this.currentCard = this.deck.cards[this.turns - 1];    
+  }
+
+  returnCurrentCard() {
+    return this.currentCard;
+  }
+
+  takeTurn(guess) {    
+    this.turns++;
+    this.currentCard = this.deck.cards[this.turns];    
+    return new Turn(guess, this.currentCard);    
```

Comment:
I think that you may want to create your new instance of turn and save it to a variable you can access the methods inside of the newTurn, which might come in handy as far as giving feed back and evaluating the answer.